### PR TITLE
particle container multiple renderers possible fix

### PR DESCRIPTION
**Pull request by ivanpopelyshev**
Fix for #4668

### CODE REVIEW

File: src/particles/ParticleContainer.js [hunk 1]
```diff
@@ -88,10 +88,16 @@ export default class ParticleContainer extends core.Container
         this._glBuffers = {};
 
         /**
-         * @member {number}
+         * @member {number[]}
+         * @private
+         */
+        this._bufferUpdateIDs = [];
```
Comment (bigtimebuddy) [inline src/particles/ParticleContainer.js]: Might be good to nullify this in the `destroy` method.

Comment (bigtimebuddy) [reply src/particles/ParticleContainer.js]: Also, while your add it, could you add a comment for the private var here.

**Comment by ivanpopelyshev** [comment]
@bigtimebuddy done.

**Comment by manubb** [comment]
Can this be reviewed and possibly merged, please?

**Comment by bigtimebuddy** [comment]
**Broken (dev)**
https://jsfiddle.net/bigtimebuddy/owkoye9o/

**Fixed (this PR)**
https://jsfiddle.net/bigtimebuddy/gx8nnjhd/8/

**Comment by bigtimebuddy** [comment]
Thanks @ivanpopelyshev. @manubb will be releasing this shortly!

**Pull request merged**